src/amo/reducers/users.js

```diff
@@ -204,6 +206,50 @@ export const loadUserAccount = ({
   };
 };
 
+export type DeleteUserAccountParams = {|
+  errorHandlerId: string,
+  userId: UserId,
+|};
+
+type DeleteUserAccountAction = {|
+  type: typeof DELETE_USER_ACCOUNT,
+  payload: DeleteUserAccountParams,
+|};
+
+export const deleteUserAccount = (
+  { errorHandlerId, userId }: DeleteUserAccountParams
+): DeleteUserAccountAction => {
+  invariant(userId, 'userId is required');
```

Comment:
Shouldn't we also have an `invariant` for `errorHandlerId`?

Reply:
of course... sorry!